tests/Microsoft.Identity.Test.LabInfrastructure/LabUserHelper.cs

```diff
@@ -50,6 +50,13 @@ static LabUserHelper()
             IsFederatedUser = false
         };
 
+        public static UserQueryParameters MsaUserQuery => new UserQueryParameters
+        {
+            IsExternalUser = true,
+            UserContains = "MSIDLAB4_Outlook",
```

Comment:
This seems like a hack, Why can't we just request an MSA user from the lab?

Reply:
yeah...but this is how the lab is set up right now. i think they are working on changes to the api, but this is how the call needs to be made for now.

---
In reply to: [241020814](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/721#discussion_r241020814) [](ancestors = 241020814)

Reply:
See comment from other place. 

---
In reply to: [241241785](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/721#discussion_r241241785) [](ancestors = 241241785,241020814)